runtime/testy/namedSensors.csv

```diff
@@ -0,0 +1,3 @@
+two,2
```

Comment:
Note made to have dawn sanitize names.

This file is currently being used for tests, so it is in the version control. When/ if we want to have a real/dummy file, a simple rename will likely be sufficient. 

Note made to address addressing multiple sensors on same board.